Bugfix/supesc 600 guest cart is not automatically assigned to an authenticated user (#1131)
spryker/spryker-docs: _includes/pbc/all/integrate-features/202204.0/integrate-glue-api/integrate-the-cart-glue-api.md

}
```

### Configure quote creation

> If you want to create a cart for the newly authenticated customer while merging guest cart with customer cart, you can enable it by adjusting the configuration constant:

Reviewer: @Zingeon  In the phrase "you can enable it", what do you mean by "it"?

Author: @VadymSachenko the following rephrasing sounds good? `You can enable creating a cart for the newly authenticated customer while merging guest cart with customer cart by adjusting the configuration constant:`

> If you want to create a cart for the newly authenticated customer while merging guest cart with customer cart, you can enable it by adjusting the configuration constant:

Reviewer: ```suggestion
You can enable the creation of a cart for the newly authenticated customer while merging the guest cart with the customer cart by adjusting the configuration constant:
```
